### PR TITLE
Bounding box information

### PR DESCRIPTION
**Pull request by surchs**
Issue for app: #457 and #499

# Summary
- added a link to the github repo
- added a modal overlay with the readme information
- added dash-bootstrap to the requirements

# App pull request
- [ ] This is a new app
- [X] I am improving an existing app (redesigns/code "makeovers")

## About

- Playground deployment URL: https://dash-playground.plotly.host/dash-image-annotation/
- Current gallery app URL: https://dash-gallery.plotly.host/dash-image-annotation/
- Python app repository link: https://github.com/plotly/dash-sample-apps/blob/master/apps/dash-image-annotation/README.md

## Workflow

- [x] I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [ ] If the app is a redesigned and/or restyled version ofI wen an
  existing gallery app, I've summarized the changes requested in the
  appropriate Streambed issue and confirm that they have been applied.
- [x] If the app is on the Dash Gallery portal, I have added a link to
  the GitHub repository for the source code in the portal description.
- [ ] If the app is a reimplementation of a Python gallery app for the
  DashR gallery, the app in this PR mimics, as closely as possible,
  the style and functionality of the existing app.=
- [ ] I have removed *all Google Analytics code* from the app's
  `assets/` folder.

## The pre-review review

I have addressed all of the following questions:

- [x] Does everything in my code serve some purpose? (I have removed
  any dead and/or irrelevant code.)
- [x] Does everything in my code have a clear purpose? (My code is
  readable and, where it isn't, it has been commented appropriately.)]
- [ ] Am I reinventing the wheel? (I think not. There is the [layout_helper](https://github.com/plotly/dash-bio/blob/81bb6fa257febb59d7841f8c5573e7231f5a9095/tests/dashbio_demos/common/layout_helper.py#L9) module in the dash-bio apps which adds a pretty github link. But here we went with a button)

### CODE REVIEW

**Comment by emmanuelle** [comment]
Thank you for the PR! I really like the fact that you could add the two buttons in the header without too much css, which was the idea of using dbc and it seems to work out well (once you had solved the conflicts with another stylesheet).

About the modal, using the Readme is a good idea, I hadn't thought about this. Would it be possible to use a markdown file with info more specific about what users might expect when they click on the button? I think they should be told that they can draw rectangles over different types of objects, select the object class in the dropdown or in the table, and switch between different images.

Also there seems to be a small rendering glitch when opening the markdown file, every line break in the file becomes a `<br>` in the html which causes unwanted line breaks. There might be an option in `dcc.Markdown` to remove line breaks or it can be done with string parsing.

File: apps/dash-image-annotation/app.py
Comment (emmanuelle) [inline apps/dash-image-annotation/app.py:210]: What would you think of using "about" or "learn more" instead of "readme"? I think "about" has my preference but it's not a strong one.

Comment (surchs) [reply apps/dash-image-annotation/app.py:210]: Yes, both are better. I'm leaning a bit more to "learn more" as it seems more explicit to me.

**Comment by surchs** [comment]
I like the idea of a dedicated markdown file for the modal. 

I think the additional linebreaks came from me using `.readlines` rather than `.read`. Somehow `dcc.Markdown` adds newline characters between the list elements created by `.readlines`. So reading it into a single string removes this behaviour.

**Comment by emmanuelle** [comment]
Great, thanks for investigating this markdown issues. Indeed `.read` should work better. 

About the github "this branch is out-of-date with the base branch", you can merge master into your branch by clicking on "update branch" (then don't forget to pull from origin on your local branch before you push new commits).

File: apps/dash-image-annotation/assets/Howto.md
Comment (emmanuelle) [inline apps/dash-image-annotation/assets/Howto.md:16]:
```suggestion
You can continue annotating additional areas of the image, selecting a different image with the `previous` and `next`
```

**Review by emmanuelle — APPROVED** [review]
Thanks for the modifications! I just left a suggestion since I believe there's a formatting typo, but otherwise :woman_dancing: !

**Comment by surchs** [comment]
Yay :woman_dancing: ! I believe I can only merge this once you have approved the review @emmanuelle

**Review by emmanuelle — APPROVED** [review]
Ha ha yes I always forget

**Pull request merged**
